src/types.ts

```diff
@@ -83,6 +84,7 @@ export type LoggerFeatureOptions = {
    * In case you want to use different attribute names, you can use this
    * options to configure the mapping.
    */
+  componentLoader?: ComponentLoader;
```

Comment:
Why is `componentLoader` optional? It should also be on different line because comment above is related to `propertiesMapping` prop.

Reply:
fixed